spec/models/scanned_resource_spec.rb

```diff
@@ -315,6 +315,7 @@
     end
     it "marshals logical order into solr" do
       subject.logical_order.order = params
+      subject.save
```

Comment:
This works, but I admit it's pretty weird.  It also explains why it's fine in the app, because of course it saves before solrizing.